app/3.0/docs/stack-blitz.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Polymer 3.0 on StackBlitz"
+---
+
+<!-- toc -->
+
+[StackBlitz](https://stackblitz.com) is an online editor for web projects. You can use StackBlitz to prototype and preview Polymer elements.
+
+Here's a [template for Polymer 3.0 to help you get started](https://stackblitz.com/edit/start-polymer3?file=start-polymer3.js). 
+
+Please note that at the time of writing, StackBlitz requires an `index.js` and `index.html` file to be present in the root folder for all JavaScript projects. You'll need to import JavaScript modules, such as elements, from `index.js`:
+
+index.js {.caption}
+
+```
+import `my-app.js`;
+```
+
+index.html {.caption}
+```
+<!-- Include WebComponents polyfills -->
+<script src="./node_modules/@webcomponents/webcomponentsjs/webcomponents-lite.js"></script>
```

Comment:
Going to be webcomponents-bundle.js (or webcomponents-loader.js? I think we'll be able to use the loader here). /cc: @e111077 who's been working with stackblitz, @kevinpschaaf who should know stuff just because.

Reply:
I think loader is more-correct?

Reply:
I thought it was bundle?